### DISCUSSION TITLE
Freeze

### DISCUSSION THREAD

**Discussion by TiagoNL81** [ulic75/power-flow-card#94] [General]
Hi, thanks for the nice layout.

In browser mode it freeze, dan I solved this?

**Comment by ulic75** [comment]
I'm not sure I understand what mean by it freezes.  If there is a number associated with flow the dots should be moving and if the number is 0 then no dots should be shown.

**Reply by TiagoNL81** [reply]
After a while the dot is not moving and the value is doing noting. other cards will work

**Reply by TiagoNL81** [reply]
I have to add something because it concerns the raspberry pi browser
